airmozilla/base/templates/email_base.html

```diff
@@ -1,47 +1,78 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html>
-
+ 
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <title>{{ subject }}</title>
-
+ 
   <style>
     @media only screen and (min-device-width: 541px) {
       .content {
         width: 540px !important;
       }
     }
+
+    body {
+      background-color: #020202;
+    }
+
+    .content {
+      color: #ffffff;
+    }
+
+    h1,h2,h3,h4,h5,h6,p {
+      color: #ffffff;
+    }
+
+    a {
+      color: #1e77bb;
+    }
+
+    a:hover {
+      color: #2592e6;
+    }
+
+  img:link {
+    border:0px;
+  }
+  .footer {
+    text-align: center;
+  }
+
   </style>
 </head>
-
+ 
 <body>
-
+ 
 <!--[if (gte mso 9)|(IE)]>
   <table width="540" align="center" cellpadding="0" cellspacing="0" border="0">
     <tr>
       <td>
 <![endif]-->
-
+ 
         <table class="content" align="center" cellpadding="0" cellspacing="0" border="0" style="width: 100%; max-width: 540px;">
           <tr>
             <td>
+              <a href="https://air.mozilla.org/">
+                <img id="logo" src="logo-airmo-md.png"></a>
```

Comment:
I suppose it doesn't. It's actually tough to recall as I did this prior to Christmas break, though it's no longer needed. I'll remove it.